specification/ai/Azure.AI.Projects/agents/tools/tool_resources.tsp

```diff
@@ -54,6 +54,11 @@ model CodeInterpreterToolResource {
   @encodedName("application/json", "file_ids")
   @maxItems(20)
   fileIds?: string[] = #[];
+
+  /** The data sources to be used. This option is mutually exclusive with fileIds. */
+  @maxItems(20)
+  @encodedName("application/json", "data_sources")
+  dataSources?: VectorStorageDataSource[];
```

Comment:
does the docs gets generated here?

Reply:
MUHAMMAD IHSAN ALI.

Reply:
The comment on line 58 is used as a doc for class and for the variable: https://github.com/Azure/azure-sdk-for-python/blob/39fcce829c41d947b86f5b244865de4fe13ee380/sdk/ai/azure-ai-projects/azure/ai/projects/models/_models.py#L645